test/common.dart

```diff
@@ -140,6 +140,10 @@ Future wait(int milliseconds) {
   return new Future.delayed(new Duration(milliseconds: milliseconds), () {});
 }
 
+wait_async(int milliseconds) async{
```

Comment:
This shouldn't be necessary, you can `await` any method call which returns a Future (it doesn't have to be `async`)